models/userProfile.js

```diff
@@ -0,0 +1,28 @@
+const database = require('../utils/database');
+
+
+exports.GetUserProfile = function (listingPK, callback = (result) => { }) 
+{
+	var db = database.connectDatabase();
+	var query = `
+    SELECT
+	    US_PK,
+	    US_Username,
+	    US_Password,
```

Comment:
do you need to grab the password? are we going to be showing it?

Reply:
Sorry don't need to grab the password, you're right